ivadomed/evaluation.py

```diff
@@ -104,7 +104,8 @@ def evaluate(bids_df, path_output, target_suffix, eval_params):
                 imed_inference.pred_to_png(painted_list,
                                            target_list,
                                            str(path_preds.joinpath(subj_acq)),
-                                           suffix="_pred_painted.png")
+                                           suffix="_pred_painted.png",
+                                           max_value=len(target_list))
```

Comment:
```suggestion
                                           max_value=3) # painted data contain 3 float values [0.0, 1.0, 2.0, 3.0] corresponding to background, TP, FP and FN objects)
```
This ensure the same behavior as the one currently on master for "painted data".
The max value is not related to the number of class in a model.

We always want to have a max_value of "3" here because they represent the 3 possible values that may be present in painted data for True Positive, False Positive and False Negative objects respectively (+ 0 for background).
By setting "3" here, we ensure that the float values 0.0, 1.1, 2.0 and 3.0 of painted data are converted to uint values 0, 85, 170 and 255 consistently in the png file.